File: app/src/main/java/com/example/breezil/favnews/di/module/SettingsFragmentsModule.java
package com.example.breezil.favnews.di.module;

import com.example.breezil.favnews.ui.PrefFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class SettingsFragmentsModule {
    @ContributesAndroidInjector
    abstract PrefFragment contributePrefFragment();

}

File: app/src/main/java/com/fave/breezil/fave/di/module/SettingsFragmentsModule.kt
package com.fave.breezil.fave.di.module

import com.fave.breezil.fave.ui.preference.PrefFragment

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class SettingsFragmentsModule {
    @ContributesAndroidInjector
    internal abstract fun contributePrefFragment(): PrefFragment

}
